packages/terra-clinical-header/tests/nightwatch/header-spec.js

```diff
@@ -50,4 +50,11 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
     browser.url(`${browser.launchUrl}/#/tests/header-tests/no-title-right-content`);
     browser.expect.element('#headerTest--content').to.be.present;
   },
+
+  'Displays a header with large left and right contents with a large title': (browser) => {
+    browser.url(`${browser.launchUrl}/#/tests/header-tests/large-left-and-right-contents-large-title`)
+    .assert.containsText('#Header h1', 'LongTitle');
```

Comment:
We want to use nightwatch's expect syntax here, not assert.

Reply:
I'm looking at this test, it seems validating it rendered would suffice given this is test example is to verify expanding height and we have already tested the header contains each piece in the test on line 31.

Reply:
@emilyrohrbough What is the reason behind using expect, rather than assert?